server/public/partials/statblock/statBlock.html

```diff
@@ -153,8 +153,15 @@ <h4>Languages</h4>
         </p>
       </div>
       <div class="property-line">
-        <h4>Challenge</h4>
-        <p>{{creature.stats.challengeRatingStr}} ({{creature.stats.experiencePoints}} XP)</p>
+        <h4 style="opacity: 0;">Challenge</h4>
+        <div class="property-line" style="float: left; margin-left: -3.5%">
+          <h4>Challenge</h4>
+          <p>{{creature.stats.challengeRatingStr}} ({{creature.stats.experiencePoints}} XP)</p>
+        </div>
+        <div class="property-line" style="float: right; margin-right: 15%">
```

Comment:
How does this margin look on different screen sizes? Particularly mobile vs desktop.

Reply:
From what I can see on both my pc and mobile, it looks fine. But to be on the safe side, I changed the logic there a bit to reflect what's done on dndbeyond